### PR TITLE
DOCS-6092 Private Locations Configuration Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Add a header for top parameters and another header for all parameters. Ready to merge.

### Motivation

DOCS-6092, chat with @HugoPct on Slack

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by HugoPct** [comment]
let's add -- in front of all commands in All Configurations
add --help at the end and good to go

File: content/en/synthetics/private_locations/configuration.md [hunk 1]
```diff
@@ -104,6 +85,60 @@ Maximum number of tests executed in parallel.
 **Default**: `10`<br>
 Maximum number of tests fetched from Datadog.
 
+**Note**: Private Location containers output logs to stdout/stderr without saving them within the container.
```
Comment (urseberry) [inline content/en/synthetics/private_locations/configuration.md:88]:
```suggestion
**Note**: Private Location containers output logs to `stdout` and `stderr` without saving them within the container.
```

File: content/en/synthetics/private_locations/configuration.md [hunk 2]
```diff
@@ -104,6 +85,60 @@ Maximum number of tests executed in parallel.
...
+
+## All configuration options
+
+`--accessKey`
+: **Type**: String <br>
+**Default**: `none`<br>
+Access Key for Datadog API authentication.
```
Comment (urseberry) [inline content/en/synthetics/private_locations/configuration.md:95]:
```suggestion
Access key for Datadog API authentication.
```

File: content/en/synthetics/private_locations/configuration.md [hunk 3]
```diff
@@ -104,6 +85,60 @@ Maximum number of tests executed in parallel.
...
+
+`--secretAccessKey`
+: **Type**: String <br>
+**Default**: `none`<br>
+Secret Access Key for Datadog API authentication.
```
Comment (urseberry) [inline content/en/synthetics/private_locations/configuration.md:100]:
```suggestion
Secret access key for Datadog API authentication.  
```

File: content/en/synthetics/private_locations/configuration.md [hunk 4]
```diff
@@ -104,6 +85,60 @@ Maximum number of tests executed in parallel.
...
+
+`--datadogApiKey`
+: **Type**: String <br>
+**Default**: `none`<br>
+Datadog API key to send browser tests artifacts (such as screenshots).  
+ 
+`--privateKey`      
+: **Type**: Array <br>
+**Default**: `none`<br>
+Private Key used to decrypt test configurations.
```
Comment (urseberry) [inline content/en/synthetics/private_locations/configuration.md:110]:
```suggestion
Private key used to decrypt test configurations.
```

File: content/en/synthetics/private_locations/configuration.md [hunk 5]
```diff
@@ -104,6 +85,60 @@ Maximum number of tests executed in parallel.
...
+
+`--publicKey`
+: **Type**: Array <br>
+**Default**: `none`<br>
+Public Key used by Datadog to encrypt test results. Composed of `--publicKey.pem`.
```
Comment (urseberry) [inline content/en/synthetics/private_locations/configuration.md:115]:
```suggestion
Public key used by Datadog to encrypt test results. Composed of `--publicKey.pem`.
```

File: content/en/synthetics/private_locations/configuration.md [hunk 6]
```diff
@@ -104,6 +85,60 @@ Maximum number of tests executed in parallel.
...
+
+`site`
+: **Type**: String <br>
+**Default**: `datadoghq.com`<br>
+Datadog site from which the private location pulls the test configuration and pushes the test results. Your `site` is {{< region-param key="dd_site" code="true" >}}.
```
Comment (urseberry) [inline content/en/synthetics/private_locations/configuration.md:120]:
```suggestion
Datadog site from which the private location pulls the test configuration and pushes the test results. Your site is {{< region-param key="dd_site" code="true" >}}.
```

File: content/en/synthetics/private_locations/configuration.md [hunk 7]
```diff
@@ -104,54 +85,154 @@ Maximum number of tests executed in parallel.
...
-Path to JSON configuration file.
+`--blockedIPRanges`
+: **Type**: Array of Strings <br>
+**Default**: `none`<br>
+Block access to specific IPs and/or CIDR in addition, or not, to the IP ranges blocked when setting the `--enableDefaultBlockedIpRanges` parameter to `true` (for example, `--blockedIPRanges.4="127.0.0.0/8" --blockedIPRanges.6="::1/128"`.)
+
+`--enableDefaultBlockedIpRanges`
+: **Type**: Boolean <br>
+**Default**: `false`<br>
+Prevent users from creating Synthetic tests on endpoints that are using reserved IP ranges (IANA [IPv4][3] and [IPv6][4] Special-Purpose Address Registry), unless for those explicitly set with the `--allowedIPRanges` parameter.
```
Comment (urseberry) [inline content/en/synthetics/private_locations/configuration.md:200]:
```suggestion
Prevent users from creating Synthetic tests on endpoints that are using reserved IP ranges (IANA [IPv4][3] and [IPv6][4] Special-Purpose Address Registry), except for those explicitly set with the `--allowedIPRanges` parameter.
```

**Review by urseberry — APPROVED** [review]
💯

**Pull request merged**